app/controllers/concerns/conferencable.rb

```diff
@@ -0,0 +1,18 @@
+module Conferencable
+    extend ActiveSupport::Concern
```

Comment:
I don't think we need this line. We're just mixing in instance-level methods and there are no nested includes. It should work with extending the module.